<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartActiv</class>
 <widget class="QMainWindow" name="StartActiv">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0" colspan="8">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="Start_B">
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="8">
     <widget class="QCustomPlot" name="Graph_Trais" native="true"/>
    </item>
    <item row="0" column="3">
     <widget class="Line" name="line_8">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="Pause_B">
      <property name="text">
       <string>Pause</string>
      </property>
     </widget>
    </item>
    <item row="2" column="5" colspan="3">
     <widget class="QWidget" name="EXTERN" native="true">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="2" column="0" colspan="5">
        <widget class="Line" name="line_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QLabel" name="Data_Text">
         <property name="text">
          <string>Data</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="Line_Tex_1">
         <property name="text">
          <string>Frequency</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="5">
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="Line" name="line_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="Line_Tex_3">
         <property name="text">
          <string>FPS (disabled)</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="Line_Tex_2">
         <property name="text">
          <string>Trace</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="Name_Text">
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="5">
        <widget class="Line" name="line_1">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="Stat_Text">
         <property name="text">
          <string>Stat</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2" colspan="3">
        <widget class="QLCDNumber" name="NUM_1">
         <property name="intValue" stdset="0">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item row="5" column="2" colspan="3">
        <widget class="QLCDNumber" name="NUM_2">
         <property name="intValue" stdset="0">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item row="7" column="2" colspan="3">
        <widget class="QLCDNumber" name="NUM_3">
         <property name="intValue" stdset="0">
          <number>0</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="Line" name="line_7">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="8">
     <widget class="Line" name="line_6">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="4">
     <widget class="QCustomPlot" name="Graph_Vs" native="true"/>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="Save_B">
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
    <item row="0" column="7">
     <widget class="QPushButton" name="Menu_B">
      <property name="text">
       <string>Menu</string>
      </property>
     </widget>
    </item>
    <item row="0" column="6">
     <widget class="Line" name="line_9">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
